pkgs/servers/home-assistant/custom-themes/material-you-theme/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "material-you-theme";
+  version = "5.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Nerwyn";
+    repo = "material-you-theme";
+    tag = version;
+    hash = "sha256-xJXhvKwp/l08/ZWi3OcGPmCdsUiMjBDwrKz5OIpD2t8=";
+  };
+
+  npmDepsHash = "sha256-g133Je2Md4nKLZucSeM6TVEdaCsR2Ja1Aj2kf7JQk6w=";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp themes/material_you.yaml $out
+
+    runHook postInstall
+  '';
+
+  passthru.entrypoint = "material_you.yaml";
```

Comment:
IMO this isn't required like it is for custom lovelace modules.

Reply:
removed